### PR TITLE
NO-JIRA: change the customnoupgrade featureset to work as force enable and force disable

### PR DESCRIPTION
**Pull request by deads2k**
…ce disable

/assign @JoelSpeed

### CODE REVIEW

File: pkg/operator/featuregates/featuregate_controller_test.go [hunk 1]
```diff
@@ -323,20 +323,28 @@ func TestFeatureGateController_sync(t *testing.T) {
...
+					//	"Banana",   // known
+					//	"Eggplant", // known
+					//	"Five",     // known
+					//	"FoieGras", // known
+					//	"Kale",     // specific
+					//	"Lettuce",  // specific
+					//	"One",      // known
+					//	"Six",      // known
+					//	"Two",      // known
+					//).
```
Comment (JoelSpeed) [inline pkg/operator/featuregates/featuregate_controller_test.go:347]: What's this for?

Comment (deads2k) [reply pkg/operator/featuregates/featuregate_controller_test.go]: > What's this for?

cruft.  Sorry, fixed in next push.

**Comment by JoelSpeed** [comment]
/retitle NO-JIRA: change the customnoupgrade featureset to work as force enable and force disable

**Comment by deads2k** [comment]
/override ci/prow/e2e-aws-serial-techpreview

that same n etworking alert....

**Pull request merged**